contracts/wrapperContracts/WrapFeeBurner.sol

```diff
@@ -0,0 +1,227 @@
+pragma solidity 0.4.18;
+
+
+import "../ERC20Interface.sol";
+import "../FeeBurner.sol";
+import "./WrapperBase.sol";
+
+
+contract WrapFeeBurner is WrapperBase {
+
+    FeeBurner public feeBurnerContract;
+    address[] internal feeSharingWallets;
+    uint public feeSharingBps = 3000;
+
+    //knc rate range
+    struct KncPerEth {
+        uint minRate;
+        uint maxRate;
+        uint pendingMinRate;
+        uint pendingMaxRate;
+    }
+
+    KncPerEth private kncPerEth;
```

Comment:
why is kncPerEth is private while in other places (like feeSharingWallets) we use internal?
I understand that private does not let you access the variable from other contracts, even not ones derived from this contract. o is it private intentionally?

Reply:
well.
Actually it has same meaning hear. since we don't intend to inherit this contract anyway.
So I assume it better be the same.
But not sure its wort
h a new deploy